code/game/machinery/cloning.dm

```diff
@@ -165,7 +165,7 @@ GLOBAL_LIST_INIT(cloner_biomass_items, list(\
 	for(var/i=new_SE.len;i<=DNA_SE_LENGTH;i++)
```

Comment:
Let's also talk about the TODO comment at the top of this code - line 121. Who ever wrote this...Why?! WHY IS THIS CODE PASSING HEXADECIMAL?!

Why isn't this using the actual DNA code? What the heck is 
`buf.dna.UI = list(0x066,0x000,0x033,0x000,0x000,0x000,0x000,0x000,0x000,0x000,0x000,0x000,0xAF0,0x000,0x000,0x000,0x000,0x000,0x000,0x000,0x000,0x000,0x033,0x066,0x0FF,0x4DB,0x002,0x690,0x000,0x000,0x000,0x328,0x045,0x5FC,0x053,0x035,0x035,0x035)
`
I have no clue which UI values these are.

`var/list/new_SE=list(0x098,0x3E8,0x403,0x44C,0x39F,0x4B0,0x59D,0x514,0x5FC,0x578,0x5DC,0x640,0x6A4)`

What SE values are these?!

Is this it? Is THIS the problem? This spaghetti? 
```
//The return of data disks?? Just for transferring between genetics machine/cloning machine.
//TO-DO: Make the genetics machine accept them.
```
These disks aren't even used! Thanks, GOON!

Reply:
![image](https://user-images.githubusercontent.com/69871346/107568643-4cff8980-6bb5-11eb-9756-9f2b6f7ea1e4.png)

Reply:
Note: DISKS are used.
The `/disk/data/monkey` disk is not used.

Reply:
@blinkdog I'd be happy to take a closer look at these disks to clarify the code in the future. 

TL;DR: It's passing Hexadecimal directly into the SE and UI lists. That's ... okay for what's it's doing - setting up hard-coded DNA for this disk type. That's real, that's fair. So, eeeh, I went through and made it messier:

```
	buf.dna.UI = list(
		0x066,	//DNA_UI_HAIR_R
		0x000,	//DNA_UI_HAIR_G
		0x033,	//DNA_UI_HAIR_B
		0x000,	//DNA_UI_HAIR_R2
		0x000,	//DNA_UI_HAIR_G2
		0x000,	//DNA_UI_HAIR_B2
		0x000,	//DNA_UI_BEARD_R
		0x000,	//DNA_UI_BEARD_G
		0x000,	//DNA_UI_BEARD_B
		0x000,	//DNA_UI_BEARD2_R
		0x000,	//DNA_UI_BEARD2_G
		0x000,	//DNA_UI_BEARD2_B
		0xAF0,	//DNA_UI_SKIN_TONE
		0x000,	//DNA_UI_SKIN_R
		0x000,	//DNA_UI_SKIN_G
		0x000,	//DNA_UI_SKIN_B
		0x000,	//DNA_UI_HACC_R
		0x000,	//DNA_UI_HACC_G
		0x000,	//DNA_UI_HACC_B
		0x000,	//DNA_UI_HEAD_MARK_R
		0x000,	//DNA_UI_HEAD_MARK_G
		0x000,	//DNA_UI_HEAD_MARK_B
		0x033,	//DNA_UI_BODY_MARK_R
		0x066,	//DNA_UI_BODY_MARK_G
		0x0FF,	//DNA_UI_BODY_MARK_B
		0x4DB,	//DNA_UI_TAIL_MARK_R
		0x002,	//DNA_UI_TAIL_MARK_B
		0x690,	//DNA_UI_TAIL_MARK_G
		0x000,	//DNA_UI_EYES_R
		0x000,	//DNA_UI_EYES_G
		0x000,	//DNA_UI_EYES_B
		0x328,	//DNA_UI_GENDER
		0x045,	//DNA_UI_BEARD_STYLE
		0x5FC,	//DNA_UI_HAIR_STYLE
		0x053,	//DNA_UI_HACC_STYLE
		0x035,	//DNA_UI_HEAD_MARK_STYLE
		0x035,	//DNA_UI_BODY_MARK_STYLE
		0x035	//DNA_UI_TAIL_MARK_STYLE
	)
```

Reply:
I feel like I messed something up. Why would there be a tail mark style?